Core/automation/lib/python/core/date.py

```diff
@@ -109,7 +109,7 @@ def to_java_zoneddatetime(value):
         return ZonedDateTime.ofInstant(value.toInstant(), ZoneId.ofOffset("GMT", ZoneOffset.ofTotalSeconds(value.getTimezoneOffset()*60)))
     # Joda DateTime
     if isinstance(value, DateTime):
-        return value.toGregorianCalendar.toZonedDateTime
+        return value.toGregorianCalendar().toZonedDateTime()
```

Comment:
Take this one out... I submitted a PR and there were a number of methods missing (). I'll sync up after your PR is merged.

Reply:
Will it work if I make a new commit with the original file? (I don't know how to roll back commits)

Reply:
commit ccef07e748461ee59c2f08c5c0c84f661f12839a revert the changes that were made to the file.

Reply:
git commit --amend

And then push. https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things

Reply:
Thanks. Your answer was after my commit ccef07e though ...

I re-opened the issue

I will study the `git command --amend` command.

The question now is if the changes i made in commit ccef07e will do the job or if you need me to do what you proposed (the `git command --amend` stuff)

Reply:
I think it will be fine. If not, I can clean it up after merging your PR.

Reply:
Thanks. 
I believe you can select the Squash and merge option on the pull request so that the pull request's commits are squashed into a single commit.